pkg/workload/workload.go

```diff
@@ -51,6 +51,11 @@ func NewInfo(w *kueue.Workload) *Info {
 	}
 }
 
+func (i *Info) RebuildWorkload(wl *kueue.Workload) {
```

Comment:
Maybe  rename as `Update` as we do in https://github.com/kubernetes-sigs/kueue/blob/811eb5c643d95793d9d09a970f0734de5766d75a/pkg/queue/cluster_queue_impl.go#L46-L49

Reply:
done